src/test/java/net/sf/jabref/logic/importer/fileformat/BibtexImporterTest.java

```diff
@@ -121,11 +121,4 @@ public void testGetDescription() {
                         + "The metadata is not required to be read here, as this class is NOT called at --import.",
                 importer.getDescription());
     }
-
-    @Test
-    public void testRecognizesDatabaseID() throws Exception {
```

Comment:
@koppor A test file has to be used as `importer.importDatabase(...)` requires a `File`. @tobiasdiez There was already a testing method `parseRecognizesDatabaseID()`.